src/client/.eslintrc.js

```diff
@@ -5,6 +5,11 @@ module.exports = {
   },
   extends: 'airbnb',
   globals: {
```

Comment:
Do you still need this?

Reply:
yes need (https://jestjs.io/docs/en/api)

Reply:
But you don’t have globals anymore?

Reply:
lint "scolding" that they are undefined in test's files.